recipe/meta.yaml

```diff
@@ -228,6 +230,13 @@ outputs:
       license: BSD-3-Clause
       license_file: LICENSE
 
+  - name: gcc-no-conda-specs
+    requirements:
+      run:
+        - gcc
+      run_constrained:
+        - conda-gcc-specs <0.0a0
```

Comment:
If you add a test for the output here, we'll see whether that environment can be resolved.